package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polkadot-deployer",
-  "version": "1.5.3",
+  "version": "1.5.3-1",
```

Comment:
Here I would use 1.5.4 or 1.6.0 , wdyt ?

The version on master is on 2.x.x so there will be no conflict

Reply:
I think either the full path should be fixed (and we will return to normal versions) or we should update the charts to use the binary regardless of the path as long as its in the path, something like which polkadot